src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string CommandName = "dotnet-add";
+        public const string UsageText = @"Usage: dotnet add [command] [arguments]
+
+Arguments:
```

Comment:
I'm curious why the existing parser doesn't work here. Is it because we need to pass on `additional args`?